py/desispec/scripts/autocalib_fiberflat.py

```diff
@@ -4,6 +4,7 @@
 
 """
 from __future__ import absolute_import, division
+import sys
```

Comment:
for the record: this `sys` was missed because it was used for a `sys.exit` upon a fatal error condition that we don't normally encounter.  The better solution is for `main` to `return 1` instead of calling `sys.exit(1)`, i.e. let the `bin/desi_autocalib_fiberflat` wrapper call `sys.exit(main(...))`.  It probably doesn't matter in this case, but that allows other functions to call `desispec.scripts.autocalib_fiberflat.main` without getting completely booted out by a `sys.exit`.  This matters more for scripts that have both a command line wrapper and are called by the pipeline, so just noting this as a design pattern.